tox.ini

```diff
@@ -0,0 +1,41 @@
+[tox]
+minversion = 1.6
+skipdist = True
+envlist = py27,py36,py37,pep8,cover,docs
+
+[testenv]
+setenv = VIRTUAL_ENV={envdir}
+usedevelop = True
+install_command = pip install {opts} {packages}
+
+deps = -r{toxinidir}/requirements.txt
+       -r{toxinidir}/test-requirements.txt
+commands =
+  pytest {posargs}
+
+[testenv:cover]
+deps = -r{toxinidir}/requirements.txt
+       -r{toxinidir}/test-requirements.txt
+commands =
+  pytest --cov=ncclient
+
+[testenv:docs]
+deps = -r{toxinidir}/test-requirements.txt
+commands = sphinx-build -b html docs/source docs/html
+
+[testenv:pep8]
+commands =
+  flake8 {posargs} ncclient test
+
+[testenv:py36]
+deps = -r{toxinidir}/requirements-py3.txt
+       -r{toxinidir}/test-requirements-py3.txt
+
+[testenv:py37]
+deps = -r{toxinidir}/requirements-py3.txt
```

Comment:
unecessary, the files are the same, so please use just one fileset instead of the -py3.txt suffixes